Added this-week-in-os-18

emmasofiah2o opened the pull request:
- Created file,
- Added og-image

mansona commented on src/posts/2022-10-26-this-week-in-os-18.md:
```diff
@@ -0,0 +1,147 @@
+- [mainmatter/mainmatter-website-mailer]
+  [#1](https://github.com/mainmatter/mainmatter-website-mailer/pull/1) Add CI
+  setup ([@marcoow])
+- [mansona/test-allow-failure]
```
this one can be ignored (I tend to prefix things in mansona with `test` if they are just test repos)

mansona commented on src/posts/2022-10-26-this-week-in-os-18.md:
```diff
@@ -0,0 +1,147 @@
+- [mansona/test-allow-failure]
+  [#1](https://github.com/mansona/test-allow-failure/pull/1) add readme
+  ([@mansona])
+- [miragejs/miragejs] [#1064](https://github.com/miragejs/miragejs/pull/1064)
```
this can go in Javascript

emmasofiah2o commented on src/posts/2022-10-26-this-week-in-os-18.md:
```diff
@@ -0,0 +1,147 @@
+
+## Unknown
+
+- [hakimel/reveal.js] [#3305](https://github.com/hakimel/reveal.js/pull/3305)
```
@lolmaus can you help me classify this? You can find the different categories in the config file

lolmaus replied:
It's `JavaScript`!

emmasofiah2o commented on src/posts/2022-10-26-this-week-in-os-18.md:
```diff
@@ -0,0 +1,147 @@
+- [hakimel/reveal.js] [#3305](https://github.com/hakimel/reveal.js/pull/3305)
+  Gulp livereload: include subfolders to watch for changes in html and md
+  ([@lolmaus])
+- [mainmatter/mainmatter-website-mailer]
```
@marcoow should I include the PRs for this repository?

marcoow replied:
yes you can

emmasofiah2o replied:
What category should they be? Internal?

marcoow replied:
yes

emmasofiah2o commented:
@marcoow just to avoid confusion: this had already been reviewed so I am allowed to merge, but I have done a few changes since the last review. Does this mean I need to ask for a review again?

marcoow commented:
So in general, review is not about being allowed to merge, it's about being certain that there aren't mistakes that the author of the PR hasn't recognized. The process isn't about authority or anything but it's driven by the understanding that nobody will produce perfect PRs at all times and we help each other in preventing accidental mistakes from being deployed to production.

If you only made some tiny changes, I guess the previous review is still fine – if you changed sth. that's non-trivial, it might make sense to request another review (see above – do the additional changes potentially include mistakes?)

emmasofiah2o commented:
Ok gotcha, then I guess I can get a second review

marcoow commented:
It's easier if you request from the entire engineering team – much better chance of someone being able to review quick.

mansona reviewed (approved):
looks good 🎉
